webextensions/api/tabGroups.json

```diff
@@ -0,0 +1,697 @@
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "139"
+                  },
+                  "firefox_android": {
+                    "version_added": false
+                  },
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "title": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "89"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "139"
+                  },
+                  "firefox_android": {
+                    "version_added": false
+                  },
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "windowId": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "89"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "139"
+                  },
+                  "firefox_android": {
+                    "version_added": false
+                  },
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
+          }
+        },
+        "update": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "89"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "139"
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "groupId": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "89"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "139"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
```

Comment:
`webextensions.api.tabGroups.update.groupId` does not need to be documented separately, as it is a core part of the API definition.

```suggestion
```